example/gmres/gmres.hpp

```diff
@@ -50,8 +50,22 @@
 #include<KokkosBlas3_trsm.hpp>
 #include<KokkosSparse_spmv.hpp>
 
+
+////////////////////////////////////////////////////////////////////////////////
+// libstdc++ half_t overloads
+////////////////////////////////////////////////////////////////////////////////
+#if !KOKKOS_HALF_T_IS_FLOAT
```

Comment:
I'm okay with putting this here, but would it make more sense in a separate file?

Reply:
this should reside in kokkos core.

Reply:
i will leave this here for now but remove it once we add it to kokkos core.